config/v1/types_network.go

```diff
@@ -67,6 +73,17 @@ type ClusterNetworkEntry struct {
 	HostPrefix uint32 `json:"hostPrefix"`
 }
 
+// ExternalIPPolicy configures exactly which IPs are allowed for the ExternalIP
+// field in a Service. If the zero struct is supplied, then none are permitted.
+type ExternalIPPolicy struct {
+	// rejectedCIDRs is the list of disallowed CIDRs. These take precedence
+	// over allowedCIDRs.
+	RejectedCIDRs []string `json:"rejectedCIDRs,omitempty"`
+
+	// allowedCIDRs is the list of allowed CIDRs.
+	AllowedCIDRs []string `json:"allowedCIDRs,omitempty"`
```

Comment:
API-wise this is fine but it seems like `AllowCIDRs` should go first logically, and maybe not be `omitempty`?

/lgtm
other than this nitpick and the "after installation" nit above